custom/js.json

```diff
@@ -541,7 +541,8 @@
     "regular_expression_literals": "var x = /ab+c/g;",
     "string_literals": "var x = 'foo';",
     "unicode_escape_sequences": "var x = '\\u00A9';",
-    "unicode_point_escapes": "var x = '\\u{2F804}';"
+    "unicode_point_escapes": "var x = '\\u{2F804}';",
+    "using_keyword": "using x = true;"
```

Comment:
Maybe it should go under `statements` instead, like `javascript.statements.let`?

Reply:
Yes, under statements would be better.

Reply:
Apologies, I only tested it in the Chrome console.

Fixed via debfaf69b471e78029cd769e2526567af11a4eb5.